Sources/Protocols/Designable/CornerDesignable.swift

```diff
@@ -48,16 +41,11 @@ public extension CornerDesignable where Self: UICollectionViewCell {
 public extension CornerDesignable where Self: UIView {
   public func configureCornerRadius() {
     if !cornerRadius.isNaN && cornerRadius > 0 {
-      if cornerSides == .allSides {
-        layer.cornerRadius = cornerRadius
-      } else {
-        layer.cornerRadius = 0.0
-
-        // if a layer mask is specified, remove it
-        layer.mask?.removeFromSuperlayer()
-
-        layer.mask = cornerSidesLayer()
-      }
+      layer.cornerRadius = 0.0
+      
+      // if a layer mask is specified, remove it
+      layer.mask?.removeFromSuperlayer()
+      layer.mask = cornerSidesLayer()
```

Comment:
can we encapsulate the code since they are very similar?

Reply:
If that's ok with you, I will do the refactor in #529 since I'm already refactoring a bit the class there

Reply:
Done there